bwaMem.wdl

```diff
@@ -227,9 +227,12 @@ task countChunkSize{
     }
 
     meta {
-        output_meta: {
-            chunkSize: "output number of lines per chunk"
-        }
+    output_meta: {
+    chunkSize: {
+        description: "output number of lines per chunk",
+        vidarr_label: "chunkSize"
```

Comment:
This is adding a label to an output from a task, it should be to bam and bamIndex workflow outputs

Reply:
That is a good catch, **bwamem2** will need to be modified too - in few cases output_meta section was absent in the workflow section, fixed now.